modules/Kruskal_algorithm/src/Kruskal_algorithm.cpp

```diff
@@ -0,0 +1,103 @@
+// Copyright 2021 Volkova Anastasia
+
+#include <utility>
+#include <algorithm>
+#include <vector>
+#include <random>
+#include <ctime>
+
+#include "include/Kruskal_algorithm.h"
+
+Graph::Graph() : n(0) {
+    G.clear();
+    MST.clear();
+    parent.resize(0);
+}
+
+Graph::Graph(const int size) {
+    n = size;
+    parent.resize(n, 0);
+    G.clear();
+    MST.clear();
+    for (int i = 0; i < n; ++i) {
+        make_set(i);
+    }
+}
+
+Graph::Graph(const Graph& tmp) : n(tmp.get_n()),
+                                 G(tmp.get_G()),
+                                 MST(tmp.get_MST()),
+                                 parent(tmp.get_parent()) {}
```

Comment:
Внутри конструктора можем напрямую достучаться до полей класса
```suggestion
Graph::Graph(const Graph& tmp) : n(tmp.n),
                                 G(tmp.G),
                                 MST(tmp.MST),
                                 parent(tmp.parent) {}
```